src/fidesops/service/connectors/query_config.py

```diff
@@ -608,3 +608,21 @@ def dry_run_query(self) -> Optional[str]:
         if mongo_query is not None:
             return self.query_to_str(mongo_query, data)
         return None
+
+
+class SaaSQueryConfig(QueryConfig[None]):
```

Comment:
I only needed the typed_filtered_values method from the base QueryConfig. Open to recommendations on how to make this cleaner.

Reply:
I left the four methods as abstract. Once I refactored the SaaS Connector logic into the SaaSQueryConfig class it made sense why those methods need to be implemented by child classes.